pydicom/data/urls.json

```diff
@@ -1,39 +1,70 @@
-    "explicit_VR-UN.dcm": "https://zenodo.org/record/3861966/files/explicit_VR-UN.dcm?download=1",
-    "gdcm-US-ALOKA-16.dcm": "https://zenodo.org/record/3861966/files/gdcm-US-ALOKA-16.dcm?download=1",
-    "gdcm-US-ALOKA-16_big.dcm": "https://zenodo.org/record/3861966/files/gdcm-US-ALOKA-16_big.dcm?download=1",
-    "JPEG-LL.dcm": "https://zenodo.org/record/3861966/files/JPEG-LL.dcm?download=1",
-    "JPEG2000_UNC.dcm": "https://zenodo.org/record/3861966/files/JPEG2000_UNC.dcm?download=1",
-    "JPGLosslessP14SV1_1s_1f_8b.dcm": "https://zenodo.org/record/3861966/files/JPGLosslessP14SV1_1s_1f_8b.dcm?download=1",
-    "mlut_18.dcm": "https://zenodo.org/record/3861966/files/mlut_18.dcm?download=1",
-    "MR-SIEMENS-DICOM-WithOverlays.dcm": "https://zenodo.org/record/3861966/files/MR-SIEMENS-DICOM-WithOverlays.dcm?download=1",
-    "MR2_J2KI.dcm": "https://zenodo.org/record/3861966/files/MR2_J2KI.dcm?download=1",
-    "MR2_J2KR.dcm": "https://zenodo.org/record/3861966/files/MR2_J2KR.dcm?download=1",
-    "MR2_UNCI.dcm": "https://zenodo.org/record/3861966/files/MR2_UNCI.dcm?download=1",
-    "MR2_UNCR.dcm": "https://zenodo.org/record/3861966/files/MR2_UNCR.dcm?download=1",
-    "OBXXXX1A.dcm": "https://zenodo.org/record/3861966/files/OBXXXX1A.dcm?download=1",
-    "OBXXXX1A_2frame.dcm": "https://zenodo.org/record/3861966/files/OBXXXX1A_2frame.dcm?download=1",
-    "OBXXXX1A_expb.dcm": "https://zenodo.org/record/3861966/files/OBXXXX1A_expb.dcm?download=1",
-    "OBXXXX1A_expb_2frame.dcm": "https://zenodo.org/record/3861966/files/OBXXXX1A_expb_2frame.dcm?download=1",
-    "OT-PAL-8-face.dcm": "https://zenodo.org/record/3861966/files/OT-PAL-8-face.dcm?download=1",
-    "RG1_J2KI.dcm": "https://zenodo.org/record/3861966/files/RG1_J2KI.dcm?download=1",
-    "RG1_J2KR.dcm": "https://zenodo.org/record/3861966/files/RG1_J2KR.dcm?download=1",
-    "RG1_UNCI.dcm": "https://zenodo.org/record/3861966/files/RG1_UNCI.dcm?download=1",
-    "RG1_UNCR.dcm": "https://zenodo.org/record/3861966/files/RG1_UNCR.dcm?download=1",
-    "RG3_J2KI.dcm": "https://zenodo.org/record/3861966/files/RG3_J2KI.dcm?download=1",
-    "RG3_J2KR.dcm": "https://zenodo.org/record/3861966/files/RG3_J2KR.dcm?download=1",
-    "RG3_UNCI.dcm": "https://zenodo.org/record/3861966/files/RG3_UNCI.dcm?download=1",
-    "RG3_UNCR.dcm": "https://zenodo.org/record/3861966/files/RG3_UNCR.dcm?download=1",
-    "US1_J2KI.dcm": "https://zenodo.org/record/3861966/files/US1_J2KI.dcm?download=1",
-    "US1_J2KR.dcm": "https://zenodo.org/record/3861966/files/US1_J2KR.dcm?download=1",
-    "US1_UNCI.dcm": "https://zenodo.org/record/3861966/files/US1_UNCI.dcm?download=1",
-    "US1_UNCR.dcm": "https://zenodo.org/record/3861966/files/US1_UNCR.dcm?download=1",
-    "vlut_04.dcm": "https://zenodo.org/record/3861966/files/vlut_04.dcm?download=1"
+    "693_J2KR.dcm": "https://github.com/pydicom/pydicom-data/raw/master/data/693_J2KR.dcm",
```

Comment:
This is quite a neat solution. Can I recommend having the URLs point to the specific commit hash instead of master? That way, if the data files are updated, and then the URL updated here, old branches/versions of pydicom will still download the old file (and have the hash agree), and new version of pydicom will download the new file.

Reply:
> Can I recommend having the URLs point to the specific commit hash instead of master?

Hmmm, it does make sense. However, we'd have to remember to update the commit hash every time we added or changed a file in the data repo, and changing the test files has been very rare (changing an existing file has probably only ever happened a couple of times).

But, since a change _has_ to happen in the data repo anyway, I guess we can make sure there are good instructions there.  Can put instructions in the PR template I suppose.  I'll look into doing that.